src/test/scala/com/mongodb/spark/TestHelper.scala

```diff
@@ -141,10 +165,4 @@ class MongoDBDefaults extends Logging {
   lazy val adminDB: MongoDatabase = mongoClient.getDatabase("admin")
   lazy val configDB: MongoDatabase = mongoClient.getDatabase("config")
 
-  Runtime.getRuntime.addShutdownHook(new ShutdownHook())
```

Comment:
No longer required as all java / scala tests do this in their afterAll / AfterClass methods.